core/src/main/resources/reference.conf

```diff
@@ -25,6 +25,11 @@ akka-persistence-jdbc {
   #
   logicalDeletion.enable = true
 
+
+  # The tag separator to use when tagging events with more than one tag.
+  # This property affects jdbc-journal.tagSeparator and jdbc-read-journal.tagSeparator.
+  tagSeparator = ","
```

Comment:
This property was defined in two different places, but we should have only one. We should use the same on the write and read side. 

`jdbc-journal.tagSeparator` and `jdbc-read-journal.tagSeparator` are now referencing this value.